zc_common/jwt_auth/tests.py

```diff
@@ -109,6 +112,47 @@ def setUp(self):
         # Create an instance of the Permission class in child class
         self.permission_obj = None
 
+    def test_revoked_permissions(self):
```

Comment:
Yea, I'm going to agree with @hamedahmadi here. This is both very complicated, and far too abstract. I really don't want to get into situation (like this one) where we're pushing minor versions to zc_common every other day, and this test function would make that worse.